capa/capabilities/domain_ip_helpers.py

```diff
@@ -0,0 +1,110 @@
+from typing import Iterator, Tuple, List
+from pathlib import Path
+
+from capa.helpers import is_runtime_ida, get_auto_format, is_runtime_ghidra
+from capa.exceptions import UnsupportedFormatError
+from capa.features.common import FORMAT_PE, FORMAT_ELF, FORMAT_CAPE, String
+from capa.features.address import Address
+from capa.features.extractors import (
+    ida,
+    ghidra,
+    elffile,
+    viv,
+    pefile,
+    binja,
+    dnfile,
+    cape,
+)
+
+from capa.render.result_document import ResultDocument
+from capa.features.extractors.base_extractor import FeatureExtractor
+
+CD = Path(__file__).resolve().parent.parent.parent
+
+# these constants are also defined in capa.main
+# defined here to avoid a circular import
```

Comment:
Perhpas define a script with all these constants inside? It is better than having repeated code